src/main.js

```diff
@@ -0,0 +1,275 @@
+        publish: cfg.app.publishMetaData === 'true',
+        baseUrl: cfg.server.publicUrl + '/v1',
+        referencePrefix: 'aws-provisioner/v1/api.json',
+        aws: cfg.aws,
+        component: cfg.app.statsComponent,
+        drain: influx,
+      });
+
+      return router; 
+    },
+  },
+
+  // API Client for *this* working copy of the repo
+  apiClient: {
+    requires: ['cfg'],
+    setup: ({cfg}) => {
+      let baseUrl = cfg.server.publicUrl + '/v1';
+      let reference = v1.reference({baseUrl});
+      let clientClass = taskcluster.createClient(reference);
+      return new clientClass({
+        agent: require('http').globalAgent,
+        baseUrl: baseUrl,
+        credentials: cfg.taskcluster.credentials,
+      });
+    },
+  },
+
+  // Table Cleaner for testing
+  tableCleaner: {
+    requires: ['WorkerType', 'WorkerState', 'Secret'],
+    setup: async ({WorkerType, WorkerState, Secret}) => {
+      await Promise.all([
+        WorkerType.scan({}, {
+          handler: async (x) => { await x.remove(); },
+        }),
+        WorkerState.scan({}, {
+          handler: async (x) => { await x.remove(); },
+        }),
+        Secret.scan({}, {
+          handler: async (x) => { await x.remove(); },
+        }),
+      ]);
+      return true;
+    },
+  },
+
+  server: {
+    requires: ['cfg', 'api'],
+    setup: ({cfg, api}) => {
+      let app = base.app(cfg.server);
+      app.use('/v1', api);
+      return app.createServer();
+    },
+  },
+
+  influx: {
+    requires: ['cfg'],
+    setup: ({cfg}) => {
+      return new base.stats.Influx({
+        connectionString: cfg.influx.connectionString,
+        maxDelay: cfg.influx.maxDelay,
+        maxPendingPoints: cfg.influx.maxPendingPoints,
+      });
+    },
+  },
+
+  awsManager: {
+    requires: ['cfg', 'ec2', 'influx'],
+    setup: ({cfg, ec2, influx}) => {
+      return new AwsManager(
+        ec2,
+        cfg.app.id,
+        cfg.app.awsKeyPrefix,
+        cfg.app.awsInstancePubkey,
+        cfg.app.maxInstanceLife,
+        influx
+      );
+    },
+  },
+
+  provisioner: {
+    requires: ['cfg', 'awsManager', 'WorkerType', 'WorkerState', 'Secret', 'ec2', 'influx'],
+    setup: async ({cfg, awsManager, WorkerType, WorkerState, Secret, ec2, influx}) => {
+      let queue = new taskcluster.Queue({credentials: cfg.taskcluster.credentials});
+
+      let provisioner = new provision.Provisioner({
+        WorkerType: WorkerType,
+        Secret: Secret,
+        WorkerState: WorkerState,
+        queue: queue,
+        provisionerId: cfg.app.id,
+        taskcluster: cfg.taskcluster,
+        influx: influx,
+        awsManager: awsManager,
+        provisionIterationInterval: cfg.app.iterationInterval,
+        dmsApiKey: cfg.deadmanssnitch.api.key,
+        iterationSnitch: cfg.deadmanssnitch.iterationSnitch,
+      });
+
+      try {
+        provisioner.run();
+      } catch (err) {
+        debug('[alert-operator] Error: ' + err.stack || err);
```

Comment:
yup!  need to figure out the stats stuff as it relates to the biaser before I rip out lib-stats and put in lib-monitor.